chromium_src/components/omnibox/common/omnibox_features.cc

```diff
@@ -18,6 +18,7 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kRichAutocompletion, base::FEATURE_DISABLED_BY_DEFAULT},
     {kStarterPackExpansion, base::FEATURE_DISABLED_BY_DEFAULT},
     {kZeroSuggestPrefetching, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kAblateSearchProviderWarmup, base::FEATURE_ENABLED_BY_DEFAULT},
```

Comment:
Do you mind moving this so that we keep the features alphabetically sorted? It's helpful when quickly scanning through them.

Also, please add a new entry to the `FeatureDefaultsTest.DisabledFeatures` (also sorted, as much as possible) in `app/feature_defaults_unittest.cc` to test this.

Reply:
@emerick Absolutely. Adding to `FeatureDefaultsTest.EnabledFeatures` however, as I think that is the correct location. Chromium disables the feature by default, and we re-enable with an override. As such, from examining other entries in that list, it seems _Enabled_ is the proper location. Correct me if I'm wrong; that file is a bit confusing (it could use a short explainer at the top, if you ask me).

Reply:
Sorry, yes., you're right I have disabling on my mind, that's what we usually do with these!